apps/src/lib/node/ledger/shell/mod.rs

```diff
@@ -368,34 +368,34 @@ where
 {
     /// The id of the current chain
     #[allow(dead_code)]
-    chain_id: ChainId,
+    pub chain_id: ChainId,
     /// The persistent storage with write log
-    pub(super) wl_storage: WlStorage<D, H>,
+    pub wl_storage: WlStorage<D, H>,
     /// Gas meter for the current block
     gas_meter: BlockGasMeter,
     /// Byzantine validators given from ABCI++ `prepare_proposal` are stored in
     /// this field. They will be slashed when we finalize the block.
     byzantine_validators: Vec<Evidence>,
     /// Path to the base directory with DB data and configs
     #[allow(dead_code)]
-    base_dir: PathBuf,
+    pub base_dir: PathBuf,
     /// Path to the WASM directory for files used in the genesis block.
-    pub(super) wasm_dir: PathBuf,
+    pub wasm_dir: PathBuf,
     /// Information about the running shell instance
     #[allow(dead_code)]
     mode: ShellMode,
     /// VP WASM compilation cache
-    pub(super) vp_wasm_cache: VpCache<WasmCacheRwAccess>,
+    pub vp_wasm_cache: VpCache<WasmCacheRwAccess>,
     /// Tx WASM compilation cache
-    pub(super) tx_wasm_cache: TxCache<WasmCacheRwAccess>,
+    pub tx_wasm_cache: TxCache<WasmCacheRwAccess>,
     /// Taken from config `storage_read_past_height_limit`. When set, will
     /// limit the how many block heights in the past can the storage be
     /// queried for reading values.
     storage_read_past_height_limit: Option<u64>,
     /// Proposal execution tracking
     pub proposal_data: HashSet<u64>,
     /// Log of events emitted by `FinalizeBlock` ABCI calls.
-    event_log: EventLog,
+    pub event_log: EventLog,
```

Comment:
there is already a pub method to access the event log, to acquire an immutable and mutable ref (`Shell::event_log()` and `Shell::event_log_mut()`, respectively)